src/stateless/dialog_button.vue

```diff
@@ -31,6 +31,18 @@ export default {
         target: { type: String, default: '_self' },
         trackName: { type: String, required: false },
     },
+    slot (h) {
+        return 'Something'
+    },
+    variations: {
+        disabled: [false, true],
+        // loading: [false, true], We would have to disable animation to test this
```

Comment:
If we can't test this it should be left out then

Reply:
Or maybe we should disable animation when `NODE_ENV=testing`?

Reply:
Hm, generally it would probably work well but I'm not really sure that is a good idea because then the components needs to be aware of our tests...